packages/closer/pages/bookings/friends.tsx

```
@@ -0,0 +1,271 @@
+import { useRouter } from 'next/router';
+
+import { useState } from 'react';
+
+import { Button, Card, ErrorMessage, Input } from '../../components/ui';
+import Heading from '../../components/ui/Heading';
+
+import { FaPlus } from '@react-icons/all-files/fa/FaPlus';
+import { FaTrash } from '@react-icons/all-files/fa/FaTrash';
+import { NextApiRequest, NextPageContext } from 'next';
```

ESLint

[error] 'NextApiRequest' is defined but never used.

(unused-imports/no-unused-imports)